### ISSUE TITLE
g_loss

### ISSUE THREAD

**Issue by Stephenlove** [VITA-Group/TransGAN#51]
The loss function of the generator is wgangp-model?or just" g_loss = -torch.mean(fake_validity)"

**Comment by yueruchen** [comment]
just `g_loss = -torch.mean(fake_validity)`
